STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,69 @@
+# Istio Steering Committee
+
+The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing. Steering was created to allow the Techincal Oversight Committee to exclusively focus on the technical aspects of the project. 
+
+## Steering Committee Charter
+
+1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio
+2. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
+3. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository
+4. Control and delegate access to and establish processes regarding other project resources/assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
+5. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
+6. Resolve any dispute from the Technical Oversight Committee.
+
+## Steering Committee Members
+
+Seats in Steering are held by a company, not by the individual users. Currently Steering consists of members from Google & IBM. These are:
+
+#### Google
+
+* [April Nassi](https://github.com/thisisnotapril) (Google)
+* [Dan Ciruli](https://github.com/oaktowner) (Google)
+* Jennifer Lin (Google)
+* [Louis Ryan](https://github.com/louiscryan) (Google)
+* [Martin Taillefer](https://github.com/geeknoid) (Google)
+* [Sven Mawson](https://github.com/smawson) (Google)
+
+#### IBM
+* [Dan Berg](https://github.com/dcberg) (IBM)
+* [Lin Sun](https://github.com/linsun) (IBM)
+* Nilesh Patel (IBM)
+
+## Meetings
+
+The Istio Steering Committee meets weekly; currently on Tuesday mornings. 
+
+Given the nature of the discussions in Steering, meetings are not currently open to the public. Questions around governance are listed as [issues in the community repo](https://github.com/istio/community/labels/steering-governance), and we invite your feedback there. 
+
+If you wish to present to the Steering committee or have a topic you'd like Steering to review, please file an issue here or reach out via [istio-steering@](https://groups.google.com/forum/#!forum/istio-steering). 
+
+
+## General Questions
+
+*Why Are Seats Company Held?*
+
+Steering was created as the Istio project was in infancy, in order to tackle governance and overall project strategy. To date Istio 1.0 has the bulk of contributions coming from the project founders, Google & IBM. Together it was decided that having strong corporate leadership was needed for the project to ensure continued velocity. Steering was created with 9 seats held, and with 11 being planned.
```

Comment:
“Steering” alone as a word is a bit jarring (add the and committee and/or SC abbrev or some such?)